scripts/dev/run-test/fm-run-test

```diff
@@ -9,14 +11,25 @@ if [ -z "$test_name" ]; then
 fi
 
 version_str="${FM_RUN_TEST_VERSIONS:+ ($FM_RUN_TEST_VERSIONS)}"
+test_out_path="$(mktemp --tmpdir fm-XXXXX)"
+time_out_path="$(mktemp --tmpdir fm-XXXXX)"
+time_fmt='%e\t%U\t%S'
```

Comment:
[nit] In the logs it wasn't obvious what each time column represented. Perhaps a short description would help?

```
## RUN:  backend_test_bitcoind
## TIME:   243.73  138.65   17.87
## DONE: backend_test_bitcoind
```

Turns into

```
## RUN:  backend_test_bitcoind
## TIME:   243.73 (elapsed)  138.65 (user)  17.87 (system)
## DONE: backend_test_bitcoind
```

Also on that note, do we care about the user and system times? Would elapsed be enough?

Reply:
I changed it to:

<time>s <memory>B <waits>W <ctx-switches>c


I don't see a good way to document it in-line (s and B will self explanatory, but W and c are just formats from https://www.man7.org/linux/man-pages/man1/time.1.html ), but I also think not many people really need to understand them.

Reply:
Ooh neat, I like it. Small followup nit, `%M` is in Kbytes on the man page, so the `B` unit is a _bit_ confusing.